Header header
geometry_msgs/Pose pose
geometry_msgs/Wrench wrench_or_stiffness
bool[] is_wrench
duration time_from_start